docs/design/README.md

```diff
@@ -11,6 +11,7 @@ Kata Containers design documents:
 - [Host cgroups](host-cgroups.md)
 - [`Inotify` support](inotify.md)
 - [Metrics(Kata 2.0)](kata-2-0-metrics.md)
+- [Metrics(Kata 3.0)](kata-3-0-metrics.md)
```

Comment:
Golang runtime also provided in 3.0, so the difference is not the version, but the runtime, golang or rust. so the title can changed to golang runtime and rust runtime.